package.json

```diff
@@ -3,7 +3,7 @@
   "displayName": "GitHub Notifications",
   "description": "A secure, customizable, statusbar icon that notifies you about notifications on GitHub.",
   "icon": "resources/logo-128x128.png",
-  "version": "2.2.0",
```

Comment:
You shouldn't update version numbers in PRs.

Reply:
I didn't know this! Thanks :) reverting it now